test/cpp/microbenchmarks/fullstack_fixtures.h

```diff
@@ -95,6 +95,7 @@ class FullstackFixture : public BaseFixture {
   }
 
   ~FullstackFixture() override {
+    channel_.reset();
```

Comment:
Out of curiosity, why was this needed?

Historically, I think we've seen a bunch of cases where tests have caught bugs triggered by shutting down the server before the client, because we then shut down the client while it's trying to reconnect.  It would be a shame to loose that -- although in the fullness of time, I would hope we could cover most of those cases with fuzzers instead.